src/beautifiers/cljfmt/index.coffee

```diff
@@ -13,9 +13,8 @@ module.exports = class Cljfmt extends Beautifier
 
   beautify: (text, language, options) ->
     formatPath = path.resolve(__dirname, "fmt.edn")
-    cljfmt = path.resolve(__dirname, "..", "..", "..", "node_modules/.bin/cljfmt")
     @tempFile("input", text).then((filePath) =>
-      @run(cljfmt, [
+      @run("cljfmt", [
```

Comment:
This would involve:
- (x) Adding `cljfmtPath` option with a falsy default value to https://github.com/Glavin001/atom-beautify/blob/master/src/languages/clojure.coffee
- (x) Add `cljfmtPath` option support to https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/cljfmt/index.coffee and let `path.resolve(__dirname, "..", "..", "..", "node_modules/.bin/cljfmt")` be the default value.